doc/language-support.xml

```diff
@@ -196,12 +196,12 @@ you need it.</para>
 
 <para>
   Currently supported interpreters are <varname>python26</varname>, <varname>python27</varname>,
-  <varname>python32</varname>, <varname>python33</varname>, <varname>python34</varname>
+  <varname>python33</varname>, <varname>python34</varname>, <varname>python35</varname>
```

Comment:
This is a good oportunity for me to ask question about how backward compatibility works in nix. Are you abandoning python3.2 in current unstable and that's why it's ok?